### PR TITLE
Properly handle multiple exclude entries for Scala 3

### LINKED ISSUE #542: add tests for list of coverage exclude patterns and maybe fix documentation (Scala 3)
The `-coverage-exclude-files` and `-coverage-exclude-packages` are `MultiStringSetting`s 
https://github.com/scala/scala3/blob/2d0e37353defcec46206e9f0845c738286aabce5/compiler/src/dotty/tools/dotc/config/ScalaSettings.scala#L122-L123

The `,` is used as a separator, at least according to 
https://github.com/scala/scala3/blob/2d0e37353defcec46206e9f0845c738286aabce5/compiler/src/dotty/tools/dotc/config/Settings.scala#L239

README example uses `;`
https://github.com/scoverage/sbt-scoverage/blob/76a39fb1203cd417e298b97e59f744d29525bbc6/README.md?plain=1#L77-L89

Using `;` with the `2.1.0` plugin and `3.4.2` Scala does not exclude specified values. Changing it to `,` seems to work. 
I have only tried it for `coverageExcludedFiles`, but I guess it works the same for both options.

Tests seems to lack this case.

### PR DESCRIPTION
**Pull request by jozic**
Closes #542

### CODE REVIEW

**Comment by jozic** [comment]
@ckipp01 @rolandtritsch please check this out, should be quick

File: src/sbt-test/scoverage/scala3-coverage-excluded-files/build.sbt [hunk 1]
```diff
@@ -4,7 +4,7 @@ scalaVersion := "3.4.2"
 
 libraryDependencies += "org.scalameta" %% "munit" % "0.7.29" % Test
 
-coverageExcludedFiles := ".*/two/GoodCoverage"
+coverageExcludedFiles := ".*/two/GoodCoverage;.*/three/GoodCoverage"
```
Comment (rolandtritsch) [inline src/sbt-test/scoverage/scala3-coverage-excluded-files/build.sbt:7]: Do the forward slashes need to be escaped like this ... ???
```scala
coverageExcludedFiles := ".*\\/two\\/GoodCoverage;.*\\/three\\/.*"
```

**Comment by rolandtritsch** [comment]
> @ckipp01 @rolandtritsch please check this out, should be quick

Hi @jozic. Sorry. Just seeing this now. Thanks for the fix. One quick question/comment. Otherwise it looks good.

File: src/sbt-test/scoverage/scala3-coverage-excluded-files/build.sbt (hunk 1 above)
Comment (jozic) [reply src/sbt-test/scoverage/scala3-coverage-excluded-files/build.sbt:7]: Hey, not sure about slashes. I believe it works as is on Linux which is proven by tests. But maybe you would want to add a windows environment for the checks and we can see if tests pass.

And thanks for checking the PR!

**Comment by rolandtritsch** [comment]
Hi @jozic. Was just testing this (more). Found one (small) problem, where the test file is missing some tests. Will merge the fix first. Will improve the test files in a seperate PR. Thanks for the fix.

**Comment by rolandtritsch** [comment]
Hi @jozic. Just release v2.1.1 - https://github.com/scoverage/sbt-scoverage/releases/tag/v2.1.1

**Pull request merged**